frontend/src/components/managePermissions/ConfirmDialogText.vue

```diff
@@ -15,7 +15,7 @@ const props = defineProps({
 <template>
     <p>
         Are you sure you want to remove
-        <strong>{{ props.role }}</strong> access to
+        <strong>{{ props.role }}</strong> access from
```

Comment:
Hi Jota, we shouldn't hardcode it here. Because this component is used by manage regular user access in `UserDataTable` , manage delegated admin in `DelegatedAdminTable` and manage admin access in `ApplicationAdminTable`. So you want to create a new property `customMsg`, by default it's value is "access", and in the case for delegated admin, pass "privilege"